SCClassLibrary/Common/Control/ServerStatus.sc

```diff
@@ -11,43 +11,34 @@ ServerStatusWatcher {
 	var <avgCPU, <peakCPU;
 	var <sampleRate, <actualSampleRate;
 
-	var reallyDeadCount = 0;
-	var <>bootNotifyFirst;
-
+	var reallyDeadCount = 0, bootNotifyFirst = true;
 
 	*new { |server|
 		^super.newCopyArgs(server)
 	}
 
-	quit { |watchShutDown = true|
+	quit { |onComplete, onFailure, watchShutDown = true|
```

Comment:
This would definitely make sense if ServerStatusWatcher were an old class. But it is new in 3.8, and this is simply the second half of the refactoring. So in this case, we should perhaps consider what is the more reasonable arg order?